native/src/api256.rs

```diff
@@ -136,6 +136,19 @@ declare_types! {
             Ok(util::public_key_to_js_object(&mut cx, &derived_public_key)?.upcast())
         }
 
+        method hash256(mut cx) {
```

Comment:
You'll also want to add this method to the TypeScript types in the `index.d.ts file.